python/lsst/daf/butler/cli/butler.py

```diff
@@ -158,9 +161,20 @@ def get_command(self, ctx, name):
         if name not in commands:
             return None
         self._raiseIfDuplicateCommands(commands)
-        return _importPlugin(commands[name][0] + "." + self._cmdNameToFuncName(name))
+        module_str = commands[name][0] + "." + self._cmdNameToFuncName(name)
+        # The click.command decorator returns an instance of a class, which
+        # is something that doImport is not expecting. We add it in as an
+        # option here to appease mypy.
+        plugin = cast(types.ModuleType | type | click.Command | None, _importPlugin(module_str))
```

Comment:
Could do, although it will probably trigger other places to ignore because `doImport` is never expecting to return an instance, only types and modules. I could change `doImport` but that probably has huge ramifications to typing elsewhere. I can try seeing what the damage is if I for `_importPlugin` to change what it can return.